ramp-database/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from codecs import open
+from setuptools import setup, find_packages
+
+version = '0.0.1'
+# Get the long description from the README file
+with open('README.md', encoding='utf-8') as f:
+    long_description = f.read()
+
+DISTNAME = 'ramp-database'
+DESCRIPTION = "RAMP database model"
+MAINTAINER = 'Alexandre Boucaud'
+MAINTAINER_EMAIL = 'boucaud.alexandre@gmail.com'
+URL = 'https://github.com/paris-saclay-cds/ramp-board'
+LICENSE = 'BSD (3-clause)'
+DOWNLOAD_URL = 'https://github.com/paris-saclay-cds/ramp-board'
+
+
+if __name__ == "__main__":
+    setup(
+        name=DISTNAME,
+        version=version,
+        maintainer=MAINTAINER,
+        include_package_data=True,
+        maintainer_email=MAINTAINER_EMAIL,
+        description=DESCRIPTION,
+        license=LICENSE,
+        url=URL,
+        download_url=DOWNLOAD_URL,
+        long_description=long_description,
+        zip_safe=False,  # the package can run out of an .egg file
+        classifiers=[
+            'Intended Audience :: Science/Research',
+            'Intended Audience :: Developers',
+            'License :: OSI Approved',
+            'Programming Language :: Python',
+            'Topic :: Software Development',
+            'Topic :: Scientific/Engineering',
+            'Operating System :: Microsoft :: Windows',
+            'Operating System :: POSIX',
+            'Operating System :: Unix',
+            'Operating System :: MacOS',
+        ],
+        install_requires=[
+            'numpy',
+            'pyyaml',
+            'sqlalchemy',
+            'psycopg2-binary',
```

Comment:
What was the reason again for needing this "psycopg2-binary" instead of just "psycopg2" ?

Reply:
it is a recommandation made by the library itself when you install psycopg2 :)

Reply:
A recommendation that is contested by many :) https://github.com/psycopg/psycopg2/issues/674

Reply:
oh god !